### PR TITLE
Add new voting strategy, FrankenDAO VotingPower [frankendao-votingpower]

### PR DESCRIPTION
**Pull request by jamesdobry**
Changes proposed in this pull request:
- Add new voting strategy, FrankenDAO VotingPower

### CODE REVIEW

File: src/strategies/index.ts [hunk 1]
```diff
@@ -415,7 +415,11 @@ import * as vstaPoolStaking from './vsta-pool-staking';
 import * as jpegdLockedJpegOf from './jpegd-locked-jpeg-of';
 import * as litDaoGovernance from './lit-dao-governance';
 import * as babywealthyclub from './babywealthyclub';
+<<<<<<< HEAD
 import * as battleflyVGFLYAndStakedGFLY from './battlefly-vgfly-and-staked-gfly';
+=======
+import * as frankendaoVotingPower from './frankendao-votingpower';
+>>>>>>> 5653af6 (Add new voting strategy, FrankenDAO VotingPower)
```
Comment (ChaituVR) [inline src/strategies/index.ts:422]: @jamesdobry Can take a look these conflict markers plz :)

File: src/strategies/frankendao-votingpower/index.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    multi.call(address, options.address, 'getVotes', [address])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, votingPower]) => [
+      address,
+      parseFloat(formatUnits(votingPower, 0))
+    ])
+  );
```
Comment (ChaituVR) [inline src/strategies/frankendao-votingpower/index.ts:33]: I feel like the `contract-call` strategy should work perfectly in your case :) Can try it?

Comment (ChaituVR) [reply src/strategies/frankendao-votingpower/index.ts:33]: Or also https://snapshot.org/#/strategy/erc20-votes

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/frankendao-votingpower/index.ts (hunk 2 above)
Comment (jamesdobry) [reply src/strategies/frankendao-votingpower/index.ts:33]: Good callout, let me try these first.

**Comment by jamesdobry** [comment]
Looks like `erc20-votes` was already a voting strategy that can work for this project.

**Pull request closed without merging**